CMakeLists.txt

```
@@ -220,6 +220,8 @@ if(BML_OPENCL)
   include(FindOpenCL)
 endif()
 
+find_package(PythonInterp 3.2 REQUIRED)
```

```suggestion
include(FindPythonInterp)
if(NOT PYTHONINTERP_FOUND) 
  message(FATAL "Could not find Python interpreter")
endif()
```

As long as we find any Python interpreter the scripts we use should work.